tests/base.py

```diff
@@ -5,26 +5,60 @@
 
 """Shared resources for tests."""
 
-import os
 import unittest
+from functools import lru_cache
 from typing import Union
 
 from detection_rules.rule import TOMLRule
 from detection_rules.rule_loader import DeprecatedCollection, DeprecatedRule, RuleCollection, production_filter
 
 
+RULE_LOADER_FAIL = False
+RULE_LOADER_FAIL_MSG = None
+RULE_LOADER_FAIL_RAISED = False
```

Comment:
Can these be just class variables of `BaseRuleTest` since they're specific to that class?

Reply:
Thanks, I meant to convert those to class variables after my initial POC and forgot all about them. 👍